src/components/TeaserFront/Image.js

```diff
@@ -15,6 +15,7 @@ const styles = {
     }
   }),
   textContainer: css({
+    overflow: 'hidden',
```

Comment:
Does this have any effect? AFAIK this does nothing, if so please remove. If not add a comment in which case this helps.

Reply:
Yes, it hides unpositioned content in the stacked case on mobile.

Without this fix:
<img width="451" alt="screen shot 2018-06-11 at 17 12 48" src="https://user-images.githubusercontent.com/23520051/41240865-cd7bf83e-6d9b-11e8-9447-acfa22b0b537.png">

With this fix:
<img width="452" alt="screen shot 2018-06-11 at 17 13 31" src="https://user-images.githubusercontent.com/23520051/41240883-d862bb34-6d9b-11e8-85ad-fb3426f37705.png">